modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -44,7 +47,7 @@ const Diagram = ({
 	const [changedScale, setChangedScale] = useState(false);
 	const [scale, setScale] = useState(1);
 	const [selectedOption, setSelectedOption] = useState(1);
-	const [cPins, setCpins] = useState(pins);
+	const [cPins, setCpins] = useState(null);
```

Comment:
I think you can initialize the state with the default value for a given state type. In this case, I guess `cPins` is an array, right? So, you can initialize with `	const [cPins, setCpins] = useState([]);`. However, use carefully because `[]` is not a "falsy" value.

```js
> !![]
> true
```